runtime/doc/job_control.txt

```diff
@@ -102,23 +102,23 @@ function. Here's a more object-oriented version of the above:
 >
     let Shell = {}
 
-    function Shell.on_stdout(job_id, data)
+    function Shell.on_stdout(job_id, data) dict
```

Comment:
I guess each of these handlers needs 3 parameters, too, so the doc is still wrong.